indico/modules/bootstrap/controllers.py

```diff
@@ -44,14 +47,26 @@ class RHBootstrap(RH):
     def _process_GET(self):
         if User.query.filter_by(is_system=False).has_rows():
             return redirect(url_for_index())
+
+        try:
+            hostname = urlparse(request.url_root).hostname.strip('[]')
```

Comment:
mostly a matter of preference, but i usually use `url_parse` from `werkzeug.urls` (we use this in many other places in indico as well)